src/DropdownMenu.js

```diff
@@ -16,11 +70,19 @@ const DropdownMenu = React.createClass({
         'dropdown-menu-right': this.props.pullRight
       };
 
+    // Remove outline on focus by default
+    let style = this.props.style || {};
+    style.outline = style.outline || 'none';
+
+    // Adding tabIndex -1 to allow for focus, but not tabable
     return (
         <ul
+          tabIndex="-1"
```

Comment:
Ok, that makes sense. In that case, i'd rather not expose the style prop. We don't really expose that anywhere else.